src/exchanges/paper/PaperExchange.ts

```diff
@@ -67,20 +68,20 @@ export class PaperExchange extends Duplex implements PublicExchangeAPI, Authenti
         if (order.side !== 'buy' && order.side !== 'sell') {
             return Promise.reject(new GTTError('Order side must be either \'buy\' or \'sell\'.  Ordder side was: ' + order.side));
         }
-        let orderSize;
-        try {
-            orderSize = new BigNumber(order.size);
-            if (orderSize.lessThan(0)) { throw new Error('must be positive'); }
-        } catch (e) {
-            return Promise.reject(new GTTError('Order size must be a positive number'));
-        }
-        let orderPrice;
-        try {
-            orderPrice = new BigNumber(order.price);
-            if (orderPrice.lessThan(0)) { throw new Error('must be positive'); }
-        } catch (e) {
+
+        // util function, n must be a number (not undefined or NaN) and must be positive
+        const testOrderNum = (n: any): boolean => !n || isNaN(+n) || +n < 0;
```

Comment:
Suggest renaming testOrderNum to assertPositiveNumber() or something. You could also pass in the message number and have it throw the GTTError.